android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UrlDeviceDiscoveryService.java

```diff
@@ -80,7 +80,8 @@
   private static final int NON_LOLLIPOP_NOTIFICATION_TITLE_COLOR = Color.parseColor("#ffffff");
   private static final int NON_LOLLIPOP_NOTIFICATION_URL_COLOR = Color.parseColor("#999999");
   private static final int NON_LOLLIPOP_NOTIFICATION_SNIPPET_COLOR = Color.parseColor("#999999");
-  private static final int NOTIFICATION_PRIORITY = NotificationCompat.PRIORITY_MIN;
+  private static final int NOTIFICATION_PRIORITY_LOW = NotificationCompat.PRIORITY_MIN;
+  private static final int NOTIFICATION_PRIORITY_HIGH = NotificationCompat.PRIORITY_DEFAULT;
```

Comment:
It's confusing to call these NOTIFICATION_PRIORITY_LOW/HIGH because Android already defines PRIORITY_LOW and PRIORITY_HIGH as notification priority levels, and they aren't the same as the levels we're using.  Could you switch this to use the NotificationCompat constants directly?